barretenberg/cpp/src/barretenberg/benchmark/protogalaxy_bench/protogalaxy.bench.cpp

```diff
@@ -28,19 +29,28 @@ template <typename Flavor> void fold_one(State& state) noexcept
         MockCircuits::construct_arithmetic_circuit(builder, log2_num_gates);
         return std::make_shared<ProverInstance>(builder);
     };
+    std::vector<std::shared_ptr<Instance>> instances;
+    for (size_t i = 0; i < k + 1; ++i) {
+        instances.emplace_back(construct_instance());
```

Comment:
Mb it's worth paralellising this?

Reply:
Its outside the actual benchmark so its fine